### PR TITLE
add badge scene

### PR DESCRIPTION
**Pull request by annieyw**
image

### CODE REVIEW

File: scenes/src/app/scenes/badge/badge-scene.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  selector: 'app-badge-scene',
+  templateUrl: './badge-scene.html',
+  styleUrls: ['./badge-scene.scss']
+})
+export class BadgeScene {
+}
+
+@NgModule({
+  imports: [
+    DemoMaterialModule
```
Comment (mmalerba) [inline scenes/src/app/scenes/badge/badge-scene.ts:15]: Can't we just import the badge and icon modules? the demo one has a _lot_ of stuff

Comment (annieyw) [reply scenes/src/app/scenes/badge/badge-scene.ts:15]: I think it auto imported the wrong one oops

File: scenes/src/app/scenes/badge/badge-scene.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.scene-badge-icon {
+  color: palevioletred;
```
Comment (mmalerba) [inline scenes/src/app/scenes/badge/badge-scene.scss:2]: oh I think this is why I was trippin about the colors, what if you use a dark gray instead? maybe something close to `#333`?

**Comment by annieyw** [comment]
image

File: scenes/src/app/scenes/badge/badge-scene.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.scene-badge-icon {
+  color: #333333;
```
Comment (mmalerba) [inline scenes/src/app/scenes/badge/badge-scene.scss:2]: nit: You can use the 3 character shortcut `#333` (this shortcut works when each color channel is just the same digit twice - e.g. `#abc === #aabbcc`)

**Comment by annieyw** [comment]
Changed to `#333`

**Comment by mmalerba** [comment]
@annieyw needs rebase

**Pull request merged**